drogon_ctl/templates/model_json.csp

```diff
@@ -20,6 +20,22 @@
     //"client_encoding": "",
     //table: An array of tables to be modelized. if the array is empty, all revealed tables are modelized.
     "tables": [],
+    //convert: the value can be changed by a function call before it is stored into database (method.json) or 
+    //        is read from database (method.json)
+    "convert": {
+      "enabled": true,
+      "items":[{
+          "table": "user",
+          "column": "password",
+          "method": {
+            "row": "decrypt_password",
+            "json": "encrypt_password"
```

Comment:
Should we use 'before_read' or 'before_write' here to make it clearer?
And please add some comments here.

Reply:
Good idea, maybe following will give an even better hint?
 'after_db_read' and 'before_db_write'